app/modules/MenuCustom/MenuCustom.tsx

```
@@ -313,97 +248,7 @@ const MenuCustom = () => {
-                        Add Combo
-                      </Button>
-                    </div>
-
-                    {/* Deal Indicator */}
-                    <div className="flex items-center gap-2 pt-1">
-                      <div className="w-2 h-2 rounded-full bg-green-500 animate-pulse" />
-                      <span className="text-xs font-medium text-green-700">Limited Time Offer</span>
-                    </div>
-                  </CardContent>
-                </Card>
+                <ComboMealCard combo={combo} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)